advanced/custom_strategy/custom_strategy.py

```diff
@@ -0,0 +1,59 @@
+from sym.sdk.strategies import AccessStrategy
+import requests
+
+
+class CustomStrategy(AccessStrategy):
+    def fetch_remote_identity(self, user):
+        """This method determines how to identify this Sym user in the third party system.
+
+        For this example, we'll use the user's email address.
+        """
+        return user.email
```

Comment:
```suggestion
        output = requests.get("https://some-api.invalid/user-search", params={"email": user.email})
        return output.json()["id"]
```
I wonder if it would be better to go with something like this, which while a little more complex than the pure email example, I think may more effectively demonstrate the intended use case of (and the real power behind) the `fetch_remote_identity()` method.

Reply:
Sure! I definitely went back and forth a few times on how realistic to make this example, but given it's an advanced one, I think you're right that this is worth it.